src/block/block_open.c

```diff
@@ -115,7 +115,6 @@ __wt_block_close(WT_SESSION_IMPL *session, WT_BLOCK *block)
 {
     WT_CONNECTION_IMPL *conn;
     WT_DECL_RET;
-    uint64_t bucket, hash;
 
     conn = S2C(session);
```

Comment:
You can remove this connection now that it's no longer being used in this function.